examples/land-cover-fastai/deep_lulc_walkthrough.ipynb

```diff
@@ -473,7 +479,7 @@
     "    # png_gen('eopatch_{}'.format(idx))\n",
     "    png_gen(op.join(path_out_sampled, 'eopatch_{}'.format(idx)), inference=False)\n",
     "    print(\"Saving RGB, LULC and Inds PNG to eopatch_{} for the coming ML pipeline\".format(idx))\n",
-    "    # del 'eopatch_{}'.format(idx)\n",
+    "    del 'eopatch_{}'.format(idx)\n",
```

Comment:
This doesn't work for me. is it even necessary, since there is the `rmtree` down below?

Reply:
yeah no need for this, removed